batch/batch/driver/instance_collection/pool.py

```diff
@@ -34,6 +35,46 @@
     'Number of scheduling loop executions per pool',
     ['pool_name'],
 )
+SCHEDULING_LOOP_RUNS_UNUSED_FREE_CORES = pc.Counter(
+    'scheduling_loop_runs_unused_free_cores',
+    'Number of scheduling loop executions per pool that did not use all of the available free cores',
+    ['pool_name'],
+)
+
+CONTROL_LOOP_RUNS = pc.Counter(
+    'control_loop_runs',
+    'Number of control loop executions per pool',
+    ['pool_name'],
+)
+
+TOTAL_CORES_SCHEDULED = pc.Gauge(
+    'total_cores_scheduled', 'Number of cores scheduled in one scheduling execution loop', ['pool_name']
+)
+TOTAL_CORES_ATTEMPTED_TO_SCHEDULE = pc.Gauge(
+    'total_cores_attempted_to_schedule',
+    'Number of cores attempted to schedule in one scheduling execution loop',
+    ['pool_name'],
+)
+
+NO_QUEUE_READY_CORES = pc.Gauge(
+    'control_loop_ready_cores_no_queue', 'Number of ready cores per control loop run without a job queue', ['pool_name']
+)
+JOB_QUEUE_READY_CORES = pc.Gauge(
+    'control_loop_ready_cores_job_queue', 'Number of ready cores per control loop run with a job queue', ['pool_name']
+)
```

Comment:
Can we use different terminology here? In my mind we're already using a job queue: the list of Ready jobs in the database, the difference here being that we currently look at the *entire* job queue but the new change looks at the *head* of the job queue